### PR TITLE
[CHORE] doc reusable workflow

### PR DESCRIPTION
**Pull request by AXL270**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,36 +8,13 @@ These workflows are for internal use only. You can't use them outside @Happydemi
...
-#### add-label.yml
-
-Used to add a label "patch" when a pull request is opened
+| Name                        | Description                                           |
+| --------------------------- | ----------------------------------------------------- |
+| deployment-notification.yml | Used to notify applications deployments to slack. A private lambda is called by this workflow. |
+| outdated-dependencies-check-bundler.yml | Used to check if a ruby `Gemfile` / `Gemfile.lock` got outdated dependencies. Send a slack notification with the `bundle outdated` output. |
+| outdated-dependencies-check-yarn.yml | Used to check if a ruby `package.json` / `yarn.lock` got outdated dependencies. Send a slack notification with the `yarn outdated` output. |
+| semver-label-checker.yml | Used to check if a label is set up on a pull request. |
+| add-label.yml | Used to add a label "patch" when a pull request is opened |
```
[the remark this answers is not in the document]
Comment (AXL270) [reply README.md]: done

File: README.md [hunk 2]
```diff
@@ -8,36 +8,13 @@ These workflows are for internal use only. You can't use them outside @Happydemi
 
 ## Workflows
 
-### Testing
-
-You can use [this repository](https://github.com/happydemics/gh-reusable-workflows-tests)
```
[the remark this answers is not in the document]
Comment (groyoh) [reply README.md:13]: The next line is still deleted 🙈

[the remark this answers is not in the document]
Comment (AXL270) [reply README.md:13]: I put back the end of the sentance

File: README.md [hunk 3]
```diff
@@ -6,42 +6,23 @@ Visibility is set to public because reusable workflows can't be private.
...
-
-Used to notify applications deployments to slack.
-
-A private lambda is called by this workflow.
-
-#### outdated-dependencies-check-bundler.yml
+## Testing
 
-Used to check if a ruby `Gemfile` / `Gemfile.lock` got outdated dependencies.
+You can use [this repository](https://github.com/happydemics/gh-reusable-workflows-tests) to test the reusable workflows.
```
Comment (totopsy) [inline README.md:11]:
```suggestion
You can use [this repository](https://github.com/happydemics/gh-reusable-workflows-tests)
to test the reusable workflows.
```

File: README.md [hunk 4]
```diff
@@ -6,42 +6,23 @@ Visibility is set to public because reusable workflows can't be private.
...
+|-----------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------|
+| deployment-notification.yml             | Used to notify applications deployments to slack. A private lambda is called by this workflow.                                             |
+| outdated-dependencies-check-bundler.yml | Used to check if a ruby `Gemfile` / `Gemfile.lock` got outdated dependencies. Send a slack notification with the `bundle outdated` output. |
+| outdated-dependencies-check-yarn.yml    | Used to check if a ruby `package.json` / `yarn.lock` got outdated dependencies. Send a slack notification with the `yarn outdated` output. |
+| semver-label-checker.yml                | Used to check if a label is set up on a pull request.                                                                                      |
+| add-label.yml                           | Used to add a label "patch" when a pull request is opened                                                                                  |
```
Comment (totopsy) [inline README.md:21]:
```suggestion
| add-label.yml                           | Used to add a label "patch" when a pull request is opened.                                                                                 |
```

**Review by totopsy — APPROVED** [review]

**Pull request merged**